image_jointer/utils.py

```diff
@@ -13,7 +13,7 @@ def __init__(
         raise NotImplementedError("Cannot construct")
 
     @staticmethod
-    def unify_image_size(inputs: tuple[_iSize] | list[_iSize], align: PositionAlign):
+    def unify_image_size(align: PositionAlign, *inputs: Figure):
```

Comment:
After the streaming, I had noticed that this method should take `Figure | Image.Image` for `*inputs`. Because this interface should have same behavior as `ImageJointer.joint()`. 🤔